lms/envs/common.py

```diff
@@ -1221,6 +1224,12 @@
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/modules/**/*.js'))
 )
 
+proctoring_js = (
+    ['proctoring/js/models/*.js'] +
```

Comment:
JS files are hosted in edx-proctoring/edx_proctoring/static/js

I can't speak to accessibility review on instructor dashboard, but I would suspect their focus has been on student facing pages. @clrux ?